amt/charts/mps/multiTenancyMiddleware.js

```diff
@@ -0,0 +1,89 @@
+// SPDX-FileCopyrightText: (C) 2025 Intel Corporation
+    // SPDX-License-Identifier: Apache-2.0
+
+    const multiTenancyMiddleware = async (req, res, next) => {
```

Comment:
are we not missing req.tenantId = decoded tenantID. Have you verified that once you call the APIs the entries in the DB have the tenantID set ?